website/static/js/dashboard.js

```diff
@@ -1048,23 +1065,26 @@ var Breadcrumbs = {
         return m('.db-breadcrumbs', m('ul', [
             items.map(function(item, index, array){
                 if(index === array.length-1){
-                    var label = item.type === 'node' ? ' Add component' : ' Add project';
-                    var addProjectTemplate = m.component(AddProject, {
-                        buttonTemplate : m('.btn.btn-sm.text-muted[data-toggle="modal"][data-target="#addProject"]', [m('i.fa.fa-plus', { style: 'font-size: 10px;'}), label]),
-                        parentID : args.breadcrumbs()[args.breadcrumbs().length-1].data.id,
-                        modalID : 'addProject',
-                        categoryList : args.categoryList,
-                        stayCallback : function () {
-                            args.allProjectsLoaded(false);
-                            args.updateList(args.breadcrumbs()[args.breadcrumbs().length-1]);
-                        }
-                    });
+                    var addProjectTemplate = '';
+                    if (!viewOnly) {
```

Comment:
I don't think the ```if (!viewOnly)``` here is necessary based on the ```&& !viewOnly``` [here](https://github.com/caneruguz/osf.io/pull/28/files#diff-87eb874537a294457ba0c331947de9b3R1087).